draft-irtf-nwcrg-coding-and-congestion-04.xml

```diff
@@ -382,8 +382,7 @@
 			</section>
 
 			<section anchor="subsec:cc-recov-inter-in" title="FEC within the transport">
-				<t>If the two FEC and CC channels are decoupled, the endpoint may exploit different protocols for each channel. The channels may be coupled and one single protocol may be exploited. In both cases, the receiver can differentiate source packets and repair symbols. The receiver may indicate both the number of source packets received and repaired symbols that were actually useful in the recovery process of packets.</t>
-				<t>QUIC may require a specific type of frame to transport such information between receiver and sender.</t>
```

Comment:
Here I just removed this sentence that I do not think is necessary and weirdly sets the focus on QUIC